generated/nidcpower/nidcpower.proto

```diff
@@ -511,19 +431,92 @@ enum SourceMode {
   SOURCE_MODE_NIDCPOWER_VAL_SEQUENCE = 1021;
 }
 
-enum TransientResponse {
-  TRANSIENT_RESPONSE_UNSPECIFIED = 0;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_NORMAL = 1038;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_FAST = 1039;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_SLOW = 1041;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_CUSTOM = 1042;
-}
 
-enum TriggerType {
-  TRIGGER_TYPE_UNSPECIFIED = 0;
-  TRIGGER_TYPE_NIDCPOWER_VAL_NONE = 1012;
-  TRIGGER_TYPE_NIDCPOWER_VAL_DIGITAL_EDGE = 1014;
-  TRIGGER_TYPE_NIDCPOWER_VAL_SOFTWARE_EDGE = 1015;
+enum NiDCPowerViInt32Enum{
+  option allow_alias = true;
+  NI_D_C_POWER_VI_INT32_ENUM_AUTORANGE_BEHAVIOR_UNSPECIFIED = 0;
+  NI_D_C_POWER_VI_INT32_ENUM_AUTORANGE_BEHAVIOR_NIDCPOWER_VAL_RANGE_UP_TO_LIMIT_THEN_DOWN = 1107;
```

Comment:
We shouldn't be including the individual enum name, e.g. `AUTORANGE_BEHAVIOR`, in this combined enum. The 2nd part also already has `NIDCPOWER_VAL` in it, so the `NIDCPOWER_VIINT32_ENUM` prefix isn't helping. IMO, this one would ideally be named `NIDCPOWER_ATTRIBUTE_VAL_RANGE_UP_TO_LIMIT_THEN_DOWN`

Reply:
Removing the `type_enum_name` prefix `(NI_D_C_POWER_VI_INT32_ENUM)` will cause conflicts if there are two attributes of type int32 and real64, which share same enum values. In that case we duplicate same set of values in both enums. If I duplicate `NIDCPOWER_VAL_INTERNAL` value across two enums, this is the error I receive.
![image](https://user-images.githubusercontent.com/7554740/118588516-26587c80-b7bc-11eb-8ae8-3b2a2ad2d557.png)


Removing the enum name from value `(AUTORANGE_BEHAVIOR)` also has potential for conflict if we have more than two enums defined in our metadata for same set of values. For e..g we have `PowerSource` and `PowerSourceInUse` enums in dcpower which have `NIDCPOWER_VAL_INTERNAL/AUXILIARY/AUTOMATIC` values. Last value doesn't exist in `PowerSourceInUse` because it is not applicable for corresponding attribute.

That said, we could probably do something to remove NIDCPOWER from middle of enum value to remove duplication. That comes from enum values defined in enums.py though, so some sort of string manipulation would be required for this. And if users copy potential attribute value from help (wihch contains NIDCPOWER_ prefix), and search for it with intellisense they might not get a hit. Is that something we should do?

Reply:
Is this a realistic scenario? The C headers would probably not define a single constant to be used by both a `ViReal64` attribute and a `ViInt32` attribute, would it? If so, then maybe the prefix should be `NIDCPOWER_INT32_VAL` and we take only the part of the enum value that comes after the `NIDCPOWER_VAL`, resulting in this:
```proto
enum NiDCPowerInt32AttributeValue {
  option allow_alias = true;
  NIDCPOWER_INT32_VAL_UNSPECIFIED = 0;
  NIDCPOWER_INT32_VAL_RANGE_UP_TO_LIMIT_THEN_DOWN = 1107;
  ...
}
```

Reply:
I have updated PR to use `NIDCPOWER_INT32_VAL` prefix and also added name of the enum (from enums.py file) compared to example you posted above. Name of the enum is required because same enum values could be present in multiple enums. For e.g. `NIDCPOWER_VAL_INTERNAL` is present in `PowerSource` and `PowerSourceInUse` enums in [dcpower](https://github.com/ni/grpc-device/blob/main/source/codegen/metadata/nidcpower/enums.py#L521).

I was able to remove `NIDCPOWER_VAL` prefix that comes with all enum values, so the names looks shorter now. Can you take another look and let me know if we are good to go for peer review in separate PR (after some cleanup).